colosseum/declaration.py

```diff
@@ -536,10 +586,7 @@ def __str__(self):
         non_default = []
         for name in _CSS_PROPERTIES:
             try:
-                non_default.append((
-                    name.replace('_', '-'),
-                    getattr(self, '_%s' % name)
-                ))
+                non_default.append((name.replace('_', '-'), getattr(self, '_%s' % name)))
```

Comment:
I think this is now back to the original code, except with worse code formatting :-) Can we revert this diff entirely?

Reply:
Sure!

Reply:
Fixed!